src/index.ts

```diff
@@ -152,6 +152,8 @@ export default function (fastify: FastifyInstance, options: SummalyOptions, done
 			});
 		}
 
+		reply.header('Cache-Control', 'public, max-age=86400, stale-while-revalidate=43200, immutable');
```

Comment:
URLの中身がmax-ageの間でも変わり得るのでimmutableは適切ではなさそう:


```suggestion
		reply.header('Cache-Control', 'public, max-age=86400, stale-while-revalidate=43200');
```